### ISSUE TITLE
Add lots of jobs to the same JobRunner 

### ISSUE THREAD

**Issue by glennpierce** [zupzup/woddle#3]
Is it possible to add a number of tasks to a 
JobRunner that of are all the same type but with a different cron execution time and the then allow tokio to sit there running them forever ?

Something like

```
#[derive(Clone)]
struct ScriptJob {
    script: parsescript::Script,
    config: JobConfig,
}

#[async_trait]
impl Job for ScriptJob {
    async fn run(&self) {
        println!("starting  my job!");
    }

    fn get_config(&self) -> &JobConfig {
        &self.config
    }
}

let config = RunnerConfig::default().check_interval(Duration::from_secs(60));
    let mut job_runner = JobRunner::new(config);

    for (name, script) in scripts.iter() {
        println!("running: {:?}", name);

        let job_cfg = JobConfig::new("my_job", "someSyncKey").interval(Duration::from_secs(120));
        let my_job = ScriptJob { script: script.clone(), config: job_cfg };

        job_runner.add_job(my_job);
    }

    tokio::spawn(async move {
        if let Err(e) = job_runner.start().await {
            eprintln!("error: {}", e);
        }
    });
```

At the moment I can't do that due to job_runner being moved.
```

let mut job_runner = JobRunner::new(config);
    |         -------------- move occurs because `job_runner` has type `JobRunner`, which does not implement the `Copy` trait
...
307 |         job_runner.add_job(my_job);
    |         ^^^^^^^^^^ --------------- `job_runner` moved due to this method call, in previous iteration of loop
```

Thanks

**Comment by zupzup** [comment]
Hey

I just released a new version of `woddle` - 0.3.1 which now includes the following API:

```rust
    let job_runner = JobRunner::new_with_vec(config, vec![my_job, my_job2]);
```

You can check out `examples/basic_vec.rs`.

Basically, you would add your jobs to a `mut vec` and then create the job_runner with the `config` and the `jobs`.

Thanks for your input on this, this is definitely a use-case I didn't think about, but which makes total sense.

**Comment by glennpierce** [comment]
Thanks that seems to work fine.
